app/controllers/application_controller.rb

```diff
@@ -6,7 +6,7 @@ class ApplicationController < ActionController::Base
 
   def mobile_device?
     if session[:mobile_override]
-      session[:mobile_override] == "1"
+      session[:mobile_override] == '1'
```

Comment:
Sounds good to me. I guess I usually use single quotes unless I need to interpolate. 

Sent from my iPhone

> On 2015/04/03, at 6:24, nikolas <notifications@github.com> wrote:
> 
> In app/controllers/application_controller.rb:
> 
> > @@ -6,7 +6,7 @@ class ApplicationController < ActionController::Base
> >  
> >    def mobile_device?
> >      if session[:mobile_override]
> > -      session[:mobile_override] == "1"
> > +      session[:mobile_override] == '1'
> by default, rubocop prefers single-quoted strings. @mcfiredrill what do you think?
> 
> \
> Reply to this email directly or view it on GitHub.
>